doc/ADDB-User-Guide.md

```diff
@@ -0,0 +1,345 @@
+# How to use ADDB
+- Can try it on jupyter notebook on [Chameleon Trovi](https://chameleoncloud.org/experiment/share/84c1d598-699b-4e8d-bee5-6fb7f5937504)
+- Or on a Skylake CENTOS 7.9 baremetal
+
+## Part 1 - Building Motr
```

Comment:
Building motr is covered at https://github.com/Seagate/cortx-motr/blob/main/doc/Quick-Start-Guide.rst, why is this needed here? Are there some special steps needed for ADDB?